lib/debug/repl.cpp

```diff
@@ -58,6 +58,14 @@ std::optional<std::unique_ptr<Repl>> Repl::fromFile(const std::string &path)
   }
   const auto io = debugger["io"].GetObject();
 
+  if (!debugger.HasMember("sensors")) {
+    logger_.log(core::LogLevel::kFatal,
+                "Missing required field 'debugger.sensors' in configuration file at %s",
+                path.c_str());
+    return std::nullopt;
+  }
+  const auto sensors = debugger["sensors"].GetObject();
+
```

Comment:
move this to be right above the `accelerometer` check, also remove empty space after